src/components/UI/edit/pictures/AlbumGridEditor.tsx

```diff
@@ -18,8 +18,16 @@ import PictureCard from '@components/UI/edit/pictures/PictureCard.tsx';
 const AlbumGridEditor: FC = () => {
     const [items, setItems] = useState(Array.from({ length: 8 }, (_, i) => (i + 1).toString()));
     const [activeId, setActiveId] = useState<string | null>(null);
-    const sensors = useSensors(useSensor(MouseSensor), useSensor(TouchSensor));
-
+    const sensors = useSensors(useSensor(MouseSensor, {
+        activationConstraint: {
+          distance: 8,
+        },
+      }), useSensor(TouchSensor, {
+        activationConstraint: {
+          distance: 8,
+        },
+      }));
```

Comment:
changed to avoid collision when clicking a button in a drag and drop div element.